codespell_lib/data/dictionary.txt

```diff
@@ -24644,6 +24644,7 @@ jumo->jump
 jumoed->jumped
 jumpimng->jumping
 jumpt->jumped, jump,
+jumpstart->jump-start
```

Comment:
What's wrong with jumpstart?
* Oxford Learner's: [`jump-start`](https://www.oxfordlearnersdictionaries.com/definition/english/jump-start?q=jumpstart)
* Cambridge: [`jump-start`](https://dictionary.cambridge.org/fr/dictionnaire/anglais/jump-start)
* Collins: [`jumpstart`](https://www.collinsdictionary.com/dictionary/english/jumpstart)
* Longman: [`jumpstart` or `jump‧start` ](https://www.ldoceonline.com/dictionary/jumpstart)
* Merriam-Webster: [`jump-start`](https://www.merriam-webster.com/dictionary/jump-start)

Reply:
`spellcheck` in VScode highlighted it as and, when I used `codespell` in CI, it did not caught, it thus I thought it was required, checked on Merriam-webster and added as it suggested. I know see that it can be used in either way... you can disregard if you want to.

Reply:
That looks like a VS Code error.

Reply:
NP - closing  the PR.